compose/src/main/kotlin/com/yourssu/handy/compose/Chip.kt

```diff
@@ -0,0 +1,111 @@
+package com.yourssu.handy.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.yourssu.handy.compose.foundation.*
+import com.yourssu.handy.compose.icons.HandyIcons
+import com.yourssu.handy.compose.icons.line.*
+
+enum class ChipSize(
+    internal val chipHeight: Dp
+) {
+    Large(32.dp),
+    Small(24.dp)
+}
+
+enum class ChipState {
+    Unselected,
+    Selected,
+    Disabled
+}
+
+@Composable
+fun Chip(
+    text: String = "",
+    modifier: Modifier = Modifier,
+    chipState: ChipState = ChipState.Unselected,
+    chipSize: ChipSize = ChipSize.Large,
+    leadingIcon: ImageVector? = HandyIcons.Line.InfoCircle,
+    trailingIcon: ImageVector? = HandyIcons.Line.Close,
+    chipOnClick: (() -> Unit)? = null,
+    leadingIconOnClick: (() -> Unit)? = null,
+    trailingIconOnClick: (() -> Unit)? = null
+) {
+
+    val backgroundColor = remember(chipState) {
+        when (chipState) {
+            ChipState.Unselected -> ColorGray100
+            ChipState.Selected -> ColorViolet100
+            ChipState.Disabled -> ColorGray50
+        }
+    }
+
+    val textColor = remember(chipState) {
+        when (chipState) {
+            ChipState.Unselected -> ColorGray700
+            ChipState.Selected -> ColorViolet500
+            ChipState.Disabled -> ColorGray300
+        }
+    }
```

Comment:
제 생각에는 chip내부에서 state를 가지고 있는 건 좋지 않은 것 같아요
직접 지정하는 게 어떨까요?? (remember가 불필요하지 않을까 하는 의견입니다!)
```suggestion
    val backgroundColor = when (chipState) {
        ChipState.Unselected -> ColorGray100
        ChipState.Selected -> ColorViolet100
        ChipState.Disabled -> ColorGray50
    }

    val textColor = when (chipState) {
        ChipState.Unselected -> ColorGray700
        ChipState.Selected -> ColorViolet500
        ChipState.Disabled -> ColorGray300
    }
```